src/actions/actions.ts

```diff
@@ -3673,6 +3673,11 @@ abstract class MoveByScreenLine extends BaseMovement {
   value: number = 1;
 
   public async execAction(position: Position, vimState: VimState): Promise<Position | IMovement> {
+    if (vimState.currentMode === ModeName.VisualLine) {
```

Comment:
Could we have a quick comment here to say why this is necessary (or which issue it is fixing)?